RELEASE.md

```diff
@@ -0,0 +1,24 @@
+# Release Process
+
+This document describes the release process for the Container Device Interface.
+
+1. Create an issue titled `Release container-device-interface vx.y.z` with the following content:
+```
+- [ ] Create a PR with the following changes:
+    - [ ] Bump the version to the target version in:
+        - [ ] The `CurrentVersion` in `specs-go/versions.go`
+        - [ ] The `tags.cncf.io/container-device-interface` version in `schema/go.mod`
+        - [ ] (*for spec changes only*) The `tags.cncf.io/container-device-interface/specs-go` version in go.mod
+        - [ ] (*for spec changes only*) The `tags.cncf.io/container-device-interface/specs-go` version in `schema/go.mod`
+        - [ ] Run `make mod-tidy` to update versions in `cmd/**/go.mod`
+        - [ ] Run `make mod-verify` to ensure modules are up to date.
+    - [ ] (*for spec changes only*) A a description to the spec changes in `SPEC.md`
+    - [ ] Implement a `requiresV*` function for the target version in `specs-go/versions.go`.
+- [ ] Merge the PR on sufficient approval.
+- [ ] Create a `vx.y.z` tag
+- [ ] (*for spec changes only*) Create a `specs-go/vx.y.w` tag. (for the first spec version `w` will be the same as `z`)
+- [ ] Create a GitHub release form the `vx.y.z` tag.
+- [ ] (*for spec changes only*) Create a GitHub release from the `specs-go/vx.y.w` tag.
+- [ ] Close the release issue,
```

Comment:
nit: trailing comma looks weird.
über-nit: maybe we should end either all or none of the sentences with a `.`. Now it looks a bit random.